practices/handover.md

```diff
@@ -0,0 +1,112 @@
+---
+slug: handover
+stages:
+  - development
+short_description: 
+tags:
+  - efficiency
+  - sustainability
+  - knowledge sharing
+  - knowledge transfer
+  - documentation
+keywords:
+  - agile-handover
+  - knowledge transfer
+  - team building
+  - communication
+  - support 
+covered_by_dx_scanner: false
+---
+
+# Handover
+
+**TL;DR**
+
+Transferring projects from one team to another is often seen as a mammoth task.  It requires a complete transfer of roles and responsibilities. However, following set protocols, having strong communication and reciprocated motivation makes this whole process a lot smoother.
+
+![Handover](/files/project_handover.jpg)
+
```

Comment:
please, delete this line (I am unsure how much would that change the design on our website and I don't want to risk it :D))

Reply:
Hey, which lines exacty? The entire TL:DR; section?

Reply:
sorry for not being clear, I will change it myself.